src/main/java/io/confluent/kafkarest/SimpleFetcher.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.kafkarest;
+
+import io.confluent.kafkarest.simpleconsumerspool.SimpleConsumerPool;
+import kafka.javaapi.FetchRequest;
+import kafka.javaapi.FetchResponse;
+import kafka.javaapi.consumer.SimpleConsumer;
+
+public class SimpleFetcher {
```

Comment:
Can we just get rid of this class entirely? It looks like it's only used for releasing the SimpleConsumer back to the pool, but if you just inline `SimpleConsumerManager.getSimpleFetcher`, which is only used in one place, then you can track the information you need in the body of `SimpleConsumerManager.consume`.